plugins/build.gradle

```diff
@@ -1,11 +1,12 @@
 plugins {
-    id("groovy-gradle-plugin")
+    id("java-gradle-plugin")
```

Comment:
Something now implicitly configures generation of javadocs. Which results in a swarm of warnings in the build log: https://github.com/boundrates/codeartifact-gradle-plugin/actions/runs/5056855397/jobs/9074869607?pr=19

I don't think we need any javadocs here. Possibly the easiest way to keep the old behaviour is to add the following to the build script:
```
tasks.withType(Javadoc).configureEach { 
    enabled = false 
}
```